### PR TITLE
Add regex validation to prevent alpha chars in phone number field

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

The PhoneNumberField widget we're using does a great job of validating/formatting phone numbers, but had an odd "feature": You could enter alpha characters in the phone number field, and it would convert them to corresponding numbers on a phone keypad. Interesting feature, but possible not expected behaviour. 

This PR adds a little regex to check for alpha characters in the Phone number field and throws a validation error.

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -299,6 +268,30 @@ msgstr "Vous verrez une confirmation que l’alerte a été envoyée"
...
+msgid "Make changes or update information in team member accounts."
+msgstr "Modifier ou mettre à jour les renseignements dans les comptes des membres."
+
+#: about/templates/about/7_admin_accounts.html
+msgid "Delete portal accounts."
+msgstr "Supprimer des comptes."
+
+#: about/templates/about/7_admin_accounts.html
+msgid "Give log in codes to team members who are locked out of the portal."
+msgstr "Fournir des codes de connexion aux membres incapables de se connecter au portail."
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:293]: Looks like the translations command just moved these around 🤷

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2218,6 +2211,10 @@ msgstr "Votre session a expiré. Reconnectez-vous pour continuer à utiliser le
 msgid "— Scan a place registration"
 msgstr ""
 
+#: register/forms.py
+msgid "Select a type of place or event"
+msgstr "Sélectionner un type de lieu ou d’événement"
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2216]: translations command discovered this missing translation from a previous PR, so I grabbed it.

File: locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -2347,6 +2344,10 @@ msgstr "Code postal"
 msgid "Enter a valid Canadian postal code."
 msgstr "Entrez un code postal canadien valide."
 
+#: register/forms.py
+msgid "Your phone number must be valid."
+msgstr "Votre numéro de téléphone doit avoir 10 chiffres."
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2349]: Just waiting on final content for this

**Review by dorianjp — APPROVED** [review]
lgtm

**Pull request merged**